tests/cpp/test_multidevice_transformer.cpp

```diff
@@ -1000,50 +1006,51 @@ TEST_P(DistributedTransformerTest, Forward) {
   const auto options =
       at::TensorOptions().dtype(at_dtype).device(communicator_->device());
   auto x_ = at::randn({B * S, E}, options).to(at::kFloat);
+  auto ln0_w_ = at::ones(E, options).to(at::kFloat);
+  auto ln0_b_ = at::zeros(E, options).to(at::kFloat);
```

Comment:
Can this hide bugs? For example, a buggy implementation might totally forget to multiply the weight and add the bias, but will pass the comparison. 

I think a better fix is to still use random weight and bias but pass them into the reference implementation.

Reply:
Sure, this is just to match standard initialization since I'm not sure what a good range is to avoid tuning tolerances

Reply:
I'd try randn as before and check whether the test will pass. It should because this is a strict improvement. 

That said, there are several places where we don't know what a good range/distribution the activations/weights should be. That's a bigger issue we can postpone to after October.